phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexTool.java

```diff
@@ -493,6 +493,10 @@ public Job getJob() throws Exception {
                 }
                 if (useSnapshot || (!isLocalIndexBuild && pDataTable.isTransactional())) {
                     PhoenixConfigurationUtil.setCurrentScnValue(configuration, maxTimeRange);
+                    if (indexVerifyType != IndexVerifyType.NONE) {
+                        LOGGER.warn("Verification is not supported for client index rebuilds,"
```

Comment:
The error message seems wrong -- verification is supported for some client-initiated index rebuilds, just not ones using a mapper other than the server build one. If you mean that the direct mapper creates the mutations and writes them as a client, that's probably not clear to the end user.